sdk/core/azure/setup.py

```diff
@@ -9,9 +9,25 @@
 from setuptools import setup
 
 
+message = """
+
+Starting v5.0.0, the 'azure' meta-package is deprecated and cannot be installed anymore.
+Please install the service specific packages prefixed by `azure` needed for your application.
+
+Here's a non-exhaustive list of common packages:
+
+-  azure-mgmt-compute (https://pypi.python.org/pypi/azure-mgmt-compute) : Management of Virtual Machines, etc.
+-  azure-mgmt-storage (https://pypi.python.org/pypi/azure-mgmt-storage) : Management of storage accounts.
+-  azure-mgmt-resource (https://pypi.python.org/pypi/azure-mgmt-resource) : Generic package about Azure Resource Management (ARM)
+-  azure-keyvault-secrets (https://pypi.python.org/pypi/azure-keyvault-secrets) : Access to secrets in Key Vault
+-  azure-storage-blob (https://pypi.python.org/pypi/azure-storage-blob) : Access to blobs in storage accounts
+"""
+
+raise RuntimeError(message)
+
 setup(
     name='azure',
-    version='4.0.0',
+    version='5.0.0',
```

Comment:
We should probably change the classifier to

Development Status :: 7 - Inactive